### PR TITLE
ci: add github action for stale

### PR DESCRIPTION
**Pull request by Madhu-1**
This PR adds 2 changes
* Remove stale  bot configuration as it is not actively maintained
* Add new GitHub action to run stale.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Review by yati1998 — DISMISSED** [review]
LGTM

File: .github/workflows/stale.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+jobs:
+  stale:
+    runs-on: ubuntu-18.04
+    if: github.repository == 'ceph/ceph-csi'
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 30
+          days-before-issue-clone: 7
```
Comment (Rakshith-R) [inline .github/workflows/stale.yaml]:
```suggestion
          days-before-issue-close: 7
```

**Review by Rakshith-R — CHANGES_REQUESTED** [review]

**Review by humblec — APPROVED** [review]

**Review by Rakshith-R — APPROVED** [review]
Thanks, LGTM

**Comment by Madhu-1** [comment]
>GitHub App like Mergify are not allowed to merge pull request where .github/workflows is changed.

Merging this one as github action got added.

**Pull request merged**